includes/polling/ports.inc.php

```diff
@@ -111,8 +111,14 @@
 $ifmib_oids = array_merge($data_oids, $stat_oids);
```

Comment:
Absolutely nothing to do with your PR but I just noticed we define $ifmib_oids and then re-define it two lines later. Crazy!